tests/endpoints/test_upload_endpoint_pvc.py

```
@@ -415,6 +425,162 @@ def test_upload_gaussian_data(self):
+            print(f"Response text: {response.text}")
+            print(f"Response headers: {dict(response.headers)}")
+            print("==================")
+
+        # Verify successful upload
+        self.assertEqual(response.status_code, 200)
+        response_json = response.json()
+        self.assertEqual(response_json["status"], "success")
+        self.assertIn(f"{n_rows} datapoints", response_json["message"])
+
+        # Verify data was stored correctly
+        inputs, outputs, metadata = asyncio.run(ModelData(payload["model_name"]).data())
```

Ruff (0.15.7)

[warning] Unpacked variable `metadata` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)